server/routes/component.test.ts

```diff
@@ -17,9 +18,11 @@ jest.mock('../applicationInfo', () => () => ({
   branchName: 'main',
 }))
 
+const token = jwt.sign(getTokenDataMock(), 'secret')
```

Comment:
Actual token required now in these tests since the `populateCurrentUser` middleware decodes the token.